torch/_torch_docs.py

```diff
@@ -792,15 +792,29 @@ def merge_dicts(*dicts):
            r"""
 as_tensor(data, dtype=None, device=None) -> Tensor
 
-Convert the data into a `torch.Tensor`. If the data is already a `Tensor` with the same `dtype` and `device`,
-no copy will be performed, otherwise a new `Tensor` will be returned with computational graph retained if data
-`Tensor` has ``requires_grad=True``. Similarly, if the data is an ``ndarray`` of the corresponding `dtype` and
-the `device` is the cpu, no copy will be performed.
+Converts :attr:`data` into a tensor, sharing data and preserving autograd
```

Comment:
Why "tensor" and not `torch.Tensor` ?

Reply:
We don't have a firm doc policy about preferring one versus the other in the descriptions, but in this case it might make more sense to be specific about the class.